utils/ssh.py

```diff
@@ -24,7 +24,46 @@
 # Default blocking time before giving up on an ssh command execution,
 # in seconds (float)
 RUNCMD_TIMEOUT = 1200.0
-SSHResult = namedtuple("SSHResult", ["rc", "output"])
+
+
+class SSHResult(namedtuple("SSHResult", ["rc", "output"])):
+    """Allows rich comparison for more convenient testing.
+
+    Given you have ``result`` which is an instance of :py:class:`SSHResult`, you can do as follows:
+
+    .. code-block:: python
+
+        assert result  # If $?=0, then the result evaluates to a truthy value and passes the assert
+        assert result == 'installed'  # direct matching of the output value
```

Comment:
Perhaps it would be good to have an "in" to check a substring also?
The Result is the output of the func right?